pkg/skaffold/config/util.go

```diff
@@ -73,6 +73,9 @@ func readConfigFileCached(filename string) (*GlobalConfig, error) {
 			return
 		}
 		configFile, configFileErr = ReadConfigFileNoCache(filenameOrDefault)
+		if configFileErr == nil {
+			logrus.Infof("Loaded Skaffold defaults from %q", filenameOrDefault)
```

Comment:
shd we move this to debug? I usually set my `default-repo` etc in global config when demoing so that the command line is as simple as `skaffold dev` on examples. 
This will add more logs statements.

Reply:
Skaffold's default log-level is warning, so this will only appear if run with `-v info`.